Google.PowerShell/Sql/GcSqlInstanceReplicaConfigCmdlet.cs

```diff
@@ -78,10 +107,11 @@ public class NewGcSqlInstanceReplicaConfigCmdlet : GcSqlCmdlet
         /// <summary>
         /// <para type="description">
         /// Interval in milliseconds between replication heartbeats.
+        /// Defaults to 100.
```

Comment:
I assume we specify the default here because the parameter is required in the underlying API?

100ms seems far too small. There is a performance cost of the overhead of a heartbeat. I'd put this to at least 5s. Maybe as high as 20 or 30s.